bassist/bassist/parser/log_file/debs_stdout.py

```diff
@@ -17,10 +21,14 @@ def parse(self):
                 if line_number < 5: continue
 
                 parts = line.split()
+                print "parts: ", parts[2]
+                store_imports(" ".join(parts))
+
 
                 assert len(parts) > 4
                 (stat, name, vers, arch) = parts[0:4]
```

Comment:
This splits for you already. Instead of joining the parts and inserting the line, please insert each of these parts.